src/ResourceManager/RedisCache/Commands.RedisCache.Test/packages.config

```diff
@@ -1,31 +1,20 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Hyak.Common" version="1.0.3" targetFramework="net45" />
```

Comment:
Same comment here - don't remove the common dependencies in the test package. The only changes in this file should be to the versions.